ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/lvm.py

```diff
@@ -155,157 +192,64 @@ def process(self, device, results, log):
-                'p': 'pvmove',
-                'v': 'virtual',
-                'i': 'mirror or raid image',
-                'I': 'image out of sync',
-                'l': 'mirror log device',
-                'c': 'under conversion',
-                'V': 'thin volume',
-                't': 'thin pool',
-                'T': 'thin pool data',
-                'e': 'raid or thin pool metadata'}.get(att, None)
-
-    def lv_permissions(self, att):
-        # Permissions: (w)riteable, (r)ead-only, (R)ead-only activation of non-read-only volume
-        return {'w': 'writeable',
-                'r': 'read-only',
-                'R': 'read-only activation of non-read-only volume'}.get(att, None)
-
-    def lv_allocation_policy(self, att):
-        # Allocation policy: (a)nywhere, (c)ontiguous, (i)nherited, c(l)ing, (n)ormal
-        return {'a': 'anywhere',
-                'c': 'contiguous',
-                'i': 'inherited',
-                'l': 'cling',
-                'n': 'normal',
-                'A': 'anywhere (Locked)',
-                'C': 'contiguous (Locked)',
-                'I': 'inherited (Locked)',
-                'L': 'cling (Locked)',
-                'N': 'normal (Locked)'}.get(att, None)
-
-    def lv_state(self, att):
-        # State: (a)ctive, (s)uspended, (I)nvalid snapshot, invalid (S)uspended snapshot,
-        # snapshot (m)erge failed, suspended snapshot (M)erge failed,
-        # mapped (d)evice present without tables, mapped device present with (i)nactive table
-        return {'a': 'active',
-                's': 'suspended',
-                'I': 'invalid snapshot',
-                'S': 'invalid suspended snapshot',
-                'm': 'snapshot merge failed',
-                'M': 'suspended snapshot merge failed',
-                'd': 'mapped device present without tables',
-                'i': 'mapped device present with inactive table'}.get(att, None)
-
-    def lv_device(self, att):
-        # device (o)pen, (X) unknown
-        return {'o': 'open',
-                'X': 'unknown'}.get(att, None)
-
-    def lv_target_type(self, att):
-        # Target  type: (C)ache, (m)irror, (r)aid, (s)napshot, (t)hin, (u)nknown, (v)irtual
-        return {'C': 'cache',
-                'm': 'mirror',
-                'r': 'raid',
-                's': 'snapshot',
-                't': 'thin',
-                'u': 'unknown',
-                'v': 'virtual'}.get(att, None)
-
-    def lv_health(self, att):
-        # Volume Health: (p)artial, (r)efresh needed, (m)ismatches exist, (w)ritemostly, (X) unknown
-        return {'p': 'partial',
-                'r': 'refresh needed',
-                'm': 'mismatches exist',
-                'w': 'writemostly',
-                'X': 'unknown'}.get(att, None)
\ No newline at end of file
+    def makeHDMap(self, columns):
+        hd_om = ObjectMap()
+        hd_om.title = columns['disk']
+        hd_om.id = self.prepId(columns['disk'])
```

Comment:
All of these components need to have more specific ids. There's a high likelihood that we'll have multiple components on the same device with the same id without some kind of prefixing.

For example, a disk could use something like the following to avoid id conflict.

```python
hd_om.id = 'disk-{}'.format(self.prepId(columns['disk'])
```